app/src/main/res/layout/promoted_story_card.xml

```diff
@@ -45,7 +47,8 @@
         app:lessonThumbnail="@{viewModel.promotedStory.lessonThumbnail}" />
 
       <LinearLayout
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
+        android:minWidth="280dp"
```

Comment:
Is this `minWidth` actually needed after above changes? I suggest removing it and check if its still needed. 
if it is needed, please share the error screenshot as it might be related to somethjing else.

Reply:
If I remove `minWidth` UI doesn't looks good as linear layout will set width according to content. I have also tried to set `layout_width="match_parent"` but still it won't work. 

### Screenshot after removing `minWidth` from LinearLayout
![image](https://user-images.githubusercontent.com/43074241/190840009-e77af506-846c-446e-820b-5e5e02703d9e.png)

Reply:
Trying removing `minWidth` and using `android:layout_width="0dp"`.

The `LinearLayout` is inside `ConstraintLayout` and therefore `match_parent` is not used, instead `0dp` is equivalent of `match_parent`

Reply:
Thanks @rt4914 for correcting me, I have updated.